snowplow-tracker/src/main/java/com/snowplowanalytics/snowplow/tracker/payload/TrackerPayload.java

```diff
@@ -31,7 +31,8 @@
     @Override
     public void add(String key, String value) {
         if (value == null || value.isEmpty()) {
-            Logger.v(TAG, "The keys value is empty, returning without adding key: %s", key);
+            Logger.v(TAG, "The keys value is empty, removing the key: %s", key);
```

Comment:
> The keys value is empty, removing the key

I misunderstood this as the key was empty, not the associated value was empty.
I think this would be clearer given the names of the parameters:

> The value is empty, removing the key